pkgs/tools/networking/bitmask-vpn/default.nix

```diff
@@ -78,6 +78,9 @@ buildGoModule rec {
     substituteInPlace providers/vendor.conf \
       --replace "provider = riseup" "provider = ${provider}"
 
+    substituteInPlace $out/share/applications/${provider}-vpn.desktop \
+      --replace icon ${provider}-vpn
```

Comment:
`--replace Icon=icon Icon=${provider}-vpn` would be clearer, but that's me nitpicking.

Reply:
I agree it's more intention-signalling, done!